Sources/PrimerSDK/Classes/User Interface/OAuth/OAuthViewController.swift

```diff
@@ -203,9 +207,9 @@ class WebViewController: UIViewController, WKNavigationDelegate {
         }
     }
 
-    override func viewWillDisappear(_ animated: Bool) {
-        delegate?.reload()
-    }
+//    override func viewWillDisappear(_ animated: Bool) {
```

Comment:
I didn't remove it because I want to investigate further how is the reload delegate used (it's used by other services as well) and why was it used here. `viewWillDisappear` is definitely not the right place, I just don't want to forget that it was used there.